mods/ts/rules/aircraft.yaml

```diff
@@ -49,7 +49,9 @@ DSHP:
 		IdealSeparation: 1275
 		CruiseAltitude: 12c512
 		AltitudeVelocity: 256
-		FlightDynamics: VTOL, Hover, MoveIntoShroud, TakeOffOnCreation, TurnToDock
+		MoveIntoShroud: true
+		CanHover: true
```

Comment:
This is redundant because the default already has `CanHover` and `CanSlide` set to true